tests/acceptance/test_yield_vs_return.py

```diff
@@ -53,7 +53,8 @@ def _get_serialized_measurement_configuration_parameters(
 
 
 @pytest.fixture(scope="module")
-def measurement_service() -> Generator[MeasurementService, None, None]:
+def measurement_service(discovery_service_process) -> Generator[MeasurementService, None, None]:
     """Test fixture that creates and hosts a measurement service."""
+    _ = discovery_service_process
```

Comment:
Did ni-python-styleguide warn about the unused parameter?

Reply:
I ran `poetry run ni-python-styleguide lint`, no errors.

Reply:
a: assigining it to "_" (throwing it away) is referencing it, 
b: unused parens are allowed.

Remove this assignment.

Reply:
@MounikaBattu17 Why are you still explicitly referencing the unused parameter?

Reply:
This file change is outdated. I have removed the unused params(`_`). Please refer to the new change.

Reply:
Why is this still in the code?

```
@pytest.fixture(scope="module")
def measurement_service(discovery_service_process) -> Generator[MeasurementService, None, None]:
    """Test fixture that creates and hosts a measurement service."""
    discovery_service_process  # <- THIS
    with yield_vs_return_measurement.measurement_service.host_service() as service:
        yield service
```

Reply:
Sorry for the confusion. I misinterpreted wrongly and removed only the left hand side assignment. Now I have corrected.